packages/component/src/Localization/pt-PT.js

```diff
@@ -16,49 +18,69 @@ function xMinutesAgo(dateStr) {
     return `Uma hora atrás`;
   } else if (deltaInHours < 5) {
-    return `${ deltaInHours } horas atrás`;
+    return `${deltaInHours} horas atrás`;
   } else if (deltaInHours <= 24) {
     return `Hoje`;
   } else if (deltaInHours <= 48) {
     return `Ontem`;
   } else if (window.Intl) {
     return new Intl.DateTimeFormat('pt-PT').format(date);
-  } else {
-    return date.toLocaleString('pt-PT', {
-      day: '2-digit',
-      hour: '2-digit',
-      hour12: false,
-      minute: '2-digit',
-      month: '2-digit',
-      year: 'numeric',
-    });
   }
+
+  return date.toLocaleString('pt-PT', {
+    day: '2-digit',
+    hour: '2-digit',
+    hour12: false,
+    minute: '2-digit',
+    month: '2-digit',
+    year: 'numeric',
+  });
+}
+
+function botSaidSomething(avatarInitials, text, timestamp) {
+  return `Bot ${avatarInitials} disse, ${text}, ${xMinutesAgo(timestamp)}`;
+}
+
+function userSaidSomething(avatarInitials, text, timestamp) {
+  return `User ${avatarInitials} disse, ${text}, ${xMinutesAgo(timestamp)}`;
 }
 
 export default {
-  // FAILED_CONNECTION_NOTIFICATION: '',
+  FAILED_CONNECTION_NOTIFICATION: 'Não foi possível ligar.',
+  INITIAL_CONNECTION_NOTIFICATION: 'A ligar...',
+  INTERRUPTED_CONNECTION_NOTIFICATION: 'Interrupção de Rede. Reconectando...',
   // Do not localize {Retry}; it is a placeholder for "Retry". English translation should be, "Send failed. Retry."
-  SEND_FAILED_KEY: 'não foi possível enviar, {Retry}.',
-  SLOW_CONNECTION_NOTIFICATION: 'A ligação está a demorar mais tempo que o normal.',
+  RENDER_ERROR_NOTIFICATION: 'Ocorreu um erro a renderizar. Por favor verifique a consola ou contacte o developer do bot.',
+  SEND_FAILED_KEY: 'O envio falhou. {Retry}.',
+  SLOW_CONNECTION_NOTIFICATION: 'A ligação está a demorar mais tempo que o normal',
+  'Bot said something': botSaidSomething,
+  'User said something': userSaidSomething,
+  'X minutes ago': xMinutesAgo,
+  // '[File of type '%1']': '[File of type '%1']",
+  // '[Unknown Card '%1']': '[Unknown Card '%1']',
+  'Adaptive Card parse error': 'Erro ao fazer parse do Adaptive Card',
+  'Adaptive Card render error': 'Erro ao renderizar o Adaptive Card',
   'Chat': 'Chat',
-  'Download file': 'Download ficheiro',
-  'Microphone off': 'Microfone desligado',
-  'Microphone on': 'Microfone ligado',
-  'Listening…': 'A Escutar…',
-  'retry': 'repetir',
-  'Retry': '{retry}', // Please alter this value if 'Retry' at the beginning of a sentence is written differently than at the end of a sentence.
+  'Download file': 'Descarregar ficheiro',
+  'Microphone off': 'Desligar microfone',
+  'Microphone on': 'Ligar microfone',
+  'Left': 'Esquerda',
+  'Listening…': 'A escutar...',
+  'New messages': 'Novas mensagens',
+  'Right': 'Direita',
+  'retry': 'tentar novamente',
+  'Retry': 'Tentar novamente',
   'Send': 'Enviar',
-  'Sending': 'a enviar',
+  'Sending': 'A enviar',
   'Speak': 'Falar',
-  'Starting…': 'A iniciar…',
+  'Starting…': 'A iniciar...',
```

Comment:
Could you switch this back to '…' (single character)

Reply:
marked as unresolved; the ellipses is still three characters.

Reply:
Wait, I definitely need help with Github.. It shows me like I did: https://github.com/microsoft/BotFramework-WebChat/pull/2015/commits/1b62894dc0f141eb9925d5ba1c000f4f3d2d7e50

Reply:
Hmm, and the branch on my repo appears good as well... https://github.com/tiagodenoronha/BotFramework-WebChat/blob/patch-1/packages/component/src/Localization/pt-PT.js

Reply:
Okay that's odd. 

So I have some bad news, haha. There's going to be a PR this week that will cause merge conflicts with this PR. Do you mind if we put this on hold for a couple days, and I can alert you when the new PR is in? That way you can branch off of the new master and incorporate your changes, and hopefully this odd situation will resolve itself in the new PR. Does that sound ok? Sorry for the trouble!